<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetDriveClass</class>
 <widget class="QWidget" name="NetDriveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Malgun Gothic</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NetDrive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineUname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelPwd">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePsd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnLogin">
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnSignup">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogout">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NetDrive.qrc"/>
 </resources>
 <connections/>
</ui>
